cmake/templates/TBBConfig.cmake.in

```diff
@@ -74,7 +74,12 @@ foreach (_tbb_component ${TBB_FIND_COMPONENTS})
         list(APPEND TBB_IMPORTED_TARGETS TBB::${_tbb_component})
         set(TBB_${_tbb_component}_FOUND 1)
     elseif (TBB_FIND_REQUIRED AND TBB_FIND_REQUIRED_${_tbb_component})
-        message(FATAL_ERROR "Missed required Intel TBB component: ${_tbb_component}")
```

Comment:
YES!

I want to attempt to use a custom FindTBB.cmake that exists in my project in order to overcome problems with broken os installed versions of TBB.

Use case "Use system version of TBB if it can be found and configured correctly, else (disable tbb capabilites|build tbb from source|use custom FindTBB.cmake that does more brute force introspection of the system with different rules).

The current homebrew installed version of TBB uses "TBBMakeConfig.cmake" to generate a TBBConfig.cmake file with incorrect hard-coded paths to the wrong directories for finding the libraries.  If TBB is installed on the system, there is no recovery in any way.

I'm trying to get a TBB dependant tool to build and be tested on the Travis continuous integration environment.  Installing TBB from homebrew causes build failure for the reasons above.  Trying to install TBB from source fails as well because the instructions for building from source for a cmake project require access to api.github.com (which is prohibited from CI environments).

Reply:
Ok, thank you for explanation, I think this change (removal of fatal error) is ready to be integrated.